apps/sel4test-tests/src/tests/breakpoints.c

```diff
@@ -264,7 +265,9 @@ test_debug_set_data_breakpoint(struct env *env)
     return sel4test_get_result();
 }
 DEFINE_TEST(BREAKPOINT_002, "Attempt to set and trigger a data breakpoint",
-            test_debug_set_data_breakpoint, config_set(CONFIG_HARDWARE_DEBUG_API))
+            test_debug_set_data_breakpoint,
+            config_set(CONFIG_HARDWARE_DEBUG_API) &&
+            !config_set(CONFIG_SIMULATION))
```

Comment:
Can you add a comment here why this test does not work ion the simulation? I assume QEMU jst does not support this?

Reply:
Good point, will add a comment. I've only observed the problem in #43 under simulation so far. If it really is a race condition, it seems to be only triggered under simulation conditions.

If we see it elsewhere as well then we should definitely change the flag, but I haven't yet so far. In fact, the only flaky test I've seen so far on hardware is `MULTICORE0004`, which does say `(flaky)` in its description.

What is flaky in the hardware tests is mostly boards not coming up for boot or not recognising correctly that they have booted and then timing out on the boot process. Problems ranging from "network cable fell out", to dodgy earth connection messing up serial, to old cable etc. Will hopefully become stable again over time.